content/docs/command-reference/metrics/show.md

```diff
@@ -98,10 +58,60 @@ compares them with a previous version.
 
 ## Examples
 
-Examples in [add](/doc/command-reference/metrics/add),
-[modify](/doc/command-reference/metrics/modify), and
-[remove](/doc/command-reference/metrics/remove) cover most of the basic cases
-for the `dvc metrics show`.
+The basic use case shows the values in the current workspace:
+
+```dvc
+$ dvc metrics show
+      data/eval.json:
+		{
+		    AUC: 0.66729,
+		    error: 0.16982,
+		    TP: 516
+		}
```

Comment:
Shoot guys actually these examples still seem outdated i.e. not reflecting 1.0 scalar metrics. This here is a JSON array. Same throughout all the examples here and in `metrics diff`. Still we can merge this PR but we should try to address this immediately after.

Reply:
right! could you please adjust the format to:
```
$ dvc metrics show
      data/eval.json:
		    AUC: 0.66729
		    error: 0.16982
		    TP: 516
```

Reply:
OK will do. I thought the whole example was wrong because a single metrics file seems to contain several scalar values but maybe I'm just confused. Updating format...